src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/SnowplowEvent.scala

```diff
@@ -67,7 +67,7 @@ object SnowplowEvent {
     }
 
   implicit val contextsDecoder: Decoder[Contexts] = deriveDecoder[Contexts].recover {
-    case DecodingFailure(_, List(DownField("data"), DownField(_))) => Contexts(List())
+    case DecodingFailure(_, l) if l.contains(DownField("data")) => Contexts(List())
```

Comment:
I'm wondering if `data` should always be a first elem? Like:

```scala
  case DecodingFailure(_, DownField("data") :: _) => Contexts(List())
```

Not sure about it at all and it should be tested, but it feels that `l.contains(DownField("data"))` is too permissive.